### PR TITLE
Fix SHA templating when pushing to stable

### PR DESCRIPTION
**Pull request by indyfree**

### CODE REVIEW

**Comment by MarcelMue** [comment]
Can we test this somehow?

File: .circleci/config.yml [hunk 1]
```diff
@@ -52,6 +52,7 @@ jobs:
         root: .
         paths:
         - ./architect
+        - ./cert-exporter
```
Comment (indyfree) [inline .circleci/config.yml]: This will persist the (already templated) chart to the workspace, so we can use it in the publishing call.

**Comment by MarcelMue** [comment]
Also not sure if that is the correct path actually - not certain on my side.

**Comment by indyfree** [comment]
It's the same path like here: https://github.com/giantswarm/chart-operator/blob/master/.circleci/config.yml#L50
So if we suppose it correct there, should be the same path no?

**Comment by MarcelMue** [comment]
Yeah, give it a try. This is hard to test and kind of unfortunate that it gets deployed to so many places.

**Comment by indyfree** [comment]
Now publishing with the same job as `architect deploy`, where templating is executed before. PTAL :)

**Pull request merged**